### PR TITLE
docs(effects): incorrect behavior of command & steps below

### PR DESCRIPTION
**Pull request by htnpro**
## PR Type

```
[ ] Bugfix
[ ] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Build related changes
[ ] CI related changes
[x] Documentation content changes
[ ] Other... Please describe:
```

## What is the current behavior?

Incorrect behavior described below. It's not creating ``app.effects.ts`` and ``app.effects.spec.ts``.

### CODE REVIEW

File: projects/ngrx.io/content/guide/effects/install.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ yarn add @ngrx/effects
 If your project is using the Angular CLI 6+ then you can install the Effects to your project with the following `ng add` command <a href="https://angular.io/cli/add" target="_blank">(details here)</a>:
 
 ```sh
-ng add @ngrx/effects@latest
+ng add @ngrx/effects@latest --minimal false
```
Comment (timdeschryver) [inline projects/ngrx.io/content/guide/effects/install.md:24]: Hi, thanks for creating this PR!

I think it would be better to respect the default options while using the command in the docs.
So I would propose to revert this change and to remove step 3 and 4 in the section below.

Comment (htnpro) [reply projects/ngrx.io/content/guide/effects/install.md:24]: @timdeschryver Hi! Actually, I think, it's not expected behavior. How will the user understand, that he needs to create these files? 🤣 
Now a user has an idea of what he needs to do.

Comment (timdeschryver) [reply projects/ngrx.io/content/guide/effects/install.md:24]: @htnpro it's the intended behavior (#2251)

Comment (htnpro) [reply projects/ngrx.io/content/guide/effects/install.md:24]: @timdeschryver Okey, understood you. Am I closing PR?

Comment (timdeschryver) [reply projects/ngrx.io/content/guide/effects/install.md:24]: If you want, you can update the docs and remove points 3 and 4.

image

Comment (htnpro) [reply projects/ngrx.io/content/guide/effects/install.md:24]: @timdeschryver sure, I will.
